crossing-training.lic

```diff
@@ -690,6 +690,8 @@ class CrossingTraining
         end
       end
     end
+
+    fput('unhide')
```

Comment:
What do you think about making this a bput?

`bput('unhide', 'You come out of hiding', 'You slip out of hiding', 'But you are not')`